Essentials/build.gradle

```diff
@@ -23,11 +26,18 @@ shadowJar {
     dependencies {
         include (dependency('io.papermc:paperlib'))
         include (dependency('org.bstats:bstats-bukkit'))
+        include (dependency('org.spongepowered:configurate-yaml'))
+        include (dependency('org.spongepowered:configurate-core'))
+        include (dependency('org.yaml:snakeyaml'))
+        include (dependency('io.leangen.geantyref:geantyref'))
         include (project(':providers:BaseProviders'))
         include (project(':providers:PaperProvider'))
         include (project(':providers:NMSReflectionProvider'))
         include (project(':providers:1_8Provider'))
     }
     relocate 'io.papermc.lib', 'com.earth2me.essentials.paperlib'
     relocate 'org.bstats.bukkit', 'com.earth2me.essentials.metrics'
+    relocate 'org.spongepowered.configurate', 'com.earth2me.essentials.configurate'
+    relocate 'org.yaml.snakeyaml', 'com.earth2me.essentials.snakeyaml'
+    relocate 'io.leangen.geantyref', 'com.earth2me.essentials.geantyref'
```

Comment:
In my opinion, should put relocated libraries in their own special package - `com.earth2me.essentials.dep.*` or something. In the chance that Essentials actually puts something in `com.earth2me.essentials.configurate` for example, there is no chance of something accidentally conflicting.

Reply:
+1 to this, though possibly `.libs.*` instead of `.dep`